app/services/idv/actions/verify_document_status_action.rb

```diff
@@ -1,126 +0,0 @@
-module Idv
-  module Actions
-    class VerifyDocumentStatusAction < Idv::Steps::DocAuthBaseStep
-      def call
-        process_async_state(async_state)
-      end
-
-      def self.analytics_submitted_event
-        :idv_doc_auth_verify_document_status_submitted
-      end
-
-      private
-
-      def process_async_state(current_async_state)
-        form = ApiDocumentVerificationStatusForm.new(
```

Comment:
This form implementation appears to be unused now. Should it be removed?

Reply:
Added this form name to LG-9611 to make sure we don't leave it behind.